File: 완전탐색/lv1/최소직사각형.py
def solution(sizes):
    arr1 = []
    arr2 = []
    for sizeIndex, sizeEl in enumerate(sizes):
      sizeEl.sort()
      arr1.append(sizeEl[0])
      arr2.append(sizeEl[1])
    return max(arr1) * max(arr2)

File: 완전탐색/lv1/최소직사각형.js
function solution(sizes) {
  const w = [];
  const h = [];
  sizes.forEach((e) => {
    const list = e.sort((a, b) => a - b);
    w.push(list[0]);
    h.push(list[1]);
  })
  return Math.max(...w) * Math.max(...h);
}